examples/calculator.py

```diff
@@ -89,7 +89,9 @@ def press(button_id: str) -> None:
             press("c")
             press("ac")
         else:
-            press(self.NAME_MAP.get(key, key))
```

Comment:
This would blow up in `press` if you pressed "shift+tab"

Reply:
Good catch!